src/tests/integration/conftest.py

```diff
@@ -103,11 +103,18 @@ def load_taxi_data(engine):
     df.to_sql(name=table_name, con=engine, chunksize=100_000, if_exists="replace")
 
 
+def load_plot_data(engine):
+    table_name = "plot_something"
+    df = pd.DataFrame({"x": range(0, 5), "y": range(5, 10)})
+    df.to_sql(name=table_name, con=engine, chunksize=100_000, if_exists="replace")
```

Comment:
let's make this dataset smaller, say 1000 observations.  there's no need to make it this big, it'll only slow things down during testing